.travis.yml

```diff
@@ -1,11 +1,19 @@
+rvm:
+  - ruby-2.3.5-p0
```

Comment:
That's not what you have in your https://github.com/Liaison-Intl/upgrade_toolbelt/blob/master/Gemfile.lock#L75

Reply:
That's the one used in the Dockerfile.

Reply:
so what's the right version? ruby 2.3.5p376 seems the most up to date?

Reply:
I agree we should use the most up to date one.

Reply:
Can we merge this and I'll make a new PR upgrading Ruby?

Reply:
2.3.5p376 is not available on the Docker's OS. 
`apk add --no-cache ruby` installs `2.3.5-r0`.